src/components/person_card/index.js

```diff
@@ -25,37 +25,56 @@ const PersonCard = (props) => {
 
   return (
     <>
-    <div className="card">
-    <div className="top">
-       <div className="text">
-         <a className="name" href={`/faculty_detail/${props.dept}/${props.deptID}`}>{props.name}</a>
-         <h4>{props.designation}</h4>
-        </div>
+      <div className="card">
```

Comment:
Why modifying this code when no changes?

Reply:
>* The thing is there is indentation happens when I save any file. If you see nothing much is changed I only updated the informations
>* The one thing I've changed is when you have link on the faculty cards and when you will click it you will see details but what about staff we can't see their details, so there was a bug in that you just check with current version, when you will the click the staff name you will see an error I found this bug I resolved it with some conditional statement.
>* other than that I've updated from original website to current one
>* Actually after pulling changes I used npm install if any extra package is already installed when I was not working on this project, so I used npm install and run the project.
I think these are the things I've done